apps/editor/src/app/atoms/searchAndFilter/userSearch.tsx

```diff
@@ -106,17 +106,16 @@ export function UserSearch({
           key={`user-id-${resetFilterKey}`}
           placeholder={placeholder}
           block
+          name={name}
           disabled={loading || !!error}
-          data={users.map(usr => ({value: usr?.id, label: getUserLabel(usr)}))}
```

Comment:
TODO: figure out why this change happened

Reply:
ah, formData already is `users.map(usr => ({value: usr?.id, label: getUserLabel(usr)}))`